build.sh

```diff
@@ -15,6 +15,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# ===========================================================================
+# Bash functions that can be used in this script or exported by using
+# source build.sh
+
+change_java_version() {
+  local jdk=$1
+  if ((jdk)) && [[ -d /usr/local/openjdk-${jdk} ]]; then
+    export JAVA_HOME=/usr/local/openjdk-${jdk}
+    export PATH=$JAVA_HOME/bin:$PATH
+    echo "----------------------"
+    echo "Java version switched:"
+  else
+    echo "Using the current Java version:"
+  fi
+  echo "  JAVA_HOME=$JAVA_HOME"
+  echo "  PATH=$PATH"
+  java -version
```

Comment:
What should happen if the local system doesn't have java installed at all?

Reply:
It's an unrecoverable failure for _*all*_ of the targets that the root `./build.sh` can take now.

In the past, if java weren't present, a couple of the targets could proceed anyway (notably `./build.sh docker` or `./build.sh sign`).  No reason to crash them if they didn't crash before.  I'll make the fix.